### ISSUE TITLE
ML models experimentation Pt 1

### ISSUE THREAD

**Issue by ManyaBondada** [syl15/CSProject#19]
**Description:** Experiment with 3 different models for classification and sentiment analysis. Compare and contrast which models yeild  the best accuracy. 

**Acceptance Criteria:** 
- Colab NB with 3 different classification models
     - Ex, categories of experimentation could include classifcation via an LLM (BERT, GPT, etc), pre finetuned disaster classfication model from hugging face, [Scikit-learn ](https://scikit learn.org/stable/supervised_learning.html) supervised learning algorithms like logistic regression, etc
     - Graph the comparison between results to see which model performs the best
- Colab NB with sentiment analysis

**Blockers:** Raw data uploaded to Postgres

**Comments:**

**Estimated time to complete (hours):** 6
